pkg/build/builder/cmd/scmauth/sslcert.go

```diff
@@ -0,0 +1,42 @@
+package scmauth
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	SSLCertName      = "ssl-cert"
```

Comment:
this is setting a CA cert, not a client certificate... we should call this ca-certificate or something similar to clarify this is setting the CA to verify the server, not identifying the user to the server